index.js

```diff
@@ -262,8 +262,10 @@ alexa.session = function(session) {
     this.details = {
```

Comment:
Ok, I was afraid that it would be complicated.. thank you for your answer @tejashah88 !

@dblock I will implement the backwards compatible way and a paragraph in UPGRADING for now. I will keep on looking for a way to implement the warning, but I'm afraid that might be beyond my current JS skills.